Domains/4-Application/ISM/Released/IntegratedStructuralModel.02.00.00.ecschema.xml

```diff
@@ -0,0 +1,2755 @@
+            <Class class="IsmAnalysisResults" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="has results" polymorphic="false">
+            <Class class="IsmAnalysisDesign" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Results</SourceDisplayLabel>
+                <TargetDisplayLabel>Design</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="IsmDesignResults_AnalysisDesign" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..1)" roleLabel="belongs to analysis design" polymorphic="false">
+            <Class class="IsmDesignResults" />
+        </Source>
+        <Target multiplicity="(0..1)" roleLabel="has results" polymorphic="false">
+            <Class class="IsmAnalysisDesign" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Results</SourceDisplayLabel>
+                <TargetDisplayLabel>Design</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <ECRelationshipClass typeName="IsmMemberDesignCriteria_AnalysisDesign" modifier="None" strength="referencing" strengthDirection="forward">
+        <BaseClass>IsmObject_Object</BaseClass>
+        <Source multiplicity="(0..1)" roleLabel="belongs to analysis design" polymorphic="true">
+            <Class class="IsmMemberDesignCriteria" />
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="has criteria" polymorphic="true">
+            <Class class="IsmAnalysisDesign" />
+        </Target>
+        <ECCustomAttributes>
+            <RelationshipHasLabels>
+                <SourceDisplayLabel>Design criteria</SourceDisplayLabel>
+                <TargetDisplayLabel>Design</TargetDisplayLabel>
+            </RelationshipHasLabels>
+        </ECCustomAttributes>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IsmResults" modifier="Abstract" displayLabel="Results">
+        <BaseClass>bis:InformationRecordElement</BaseClass>
+        <BaseClass>IIsmSubObject</BaseClass>
+        <ECProperty propertyName="AnalysisEngine" typeName="string" displayLabel="Analysis engine name" description="Analysis engine name." />
+        <ECProperty propertyName="AnalysisFileName" typeName="string" displayLabel="File analysis was ran against" description="Analysis file name." />
+        <ECProperty propertyName="AnalyzedTime" typeName="dateTime" displayLabel="Analyzed time" description="Analyzed time." />
+        <ECProperty propertyName="AnalysisEngineVersion" typeName="string" displayLabel="Analysis engine version" description="Analysis engine version." />
```

Comment:
descriptions unnecessary, same as display label except for `AnalysisFileName` where display label and description seem swapped.

Reply:
removed